microservices/email-notification-dispatcher/dev/values.yaml

```diff
@@ -23,6 +23,7 @@ deployment:
     EMAIL_SENDER_TOPIC: "common-kafka.EMAIL_SENDER_TOPIC"
     KAFKA_BROKERS: "common-kafka.KAFKA_BROKERS"
     PRODUCER_KAFKA_CLIENT_ID: "{{.Values.name}}.KAFKA_CLIENT_ID"
+    PRODUCER_KAFKA_TRANSACTIONAL_ID: "{{.Values.namespace}}-email-notification-dispatcher-transactional-id"
```

Comment:
Setting transactional id for this service to allow publishing messages on Kafka transactionally, as per https://github.com/pagopa/interop-be-monorepo/pull/2526/files

This assumes there is a single replica for this service (if it's later scaled to multiple replicas, these should have distinct transactional ids – but for now we opted to start with a single replica, considering that AWS SES is a limiting factor on throughput anyway)